base_layer/wallet/src/transaction_service/config.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019. The Tari Project
```

Comment:
Reminder to update your Copyright notice year

Reply:
haha, will do